test/Interop/Cxx/class/initializer-from-inline.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %swift -I %S/Inputs -enable-cxx-interop -emit-ir %s | %FileCheck %s
+
+import InitializerFromInline
+
+// CHECK: define linkonce_odr i32 @{{_Z5get42v|\?get42@@YAHXZ}}
+// CHECK: define linkonce_odr i32 @{{_Z15passThroughArgTIiET_S0_|\?\?$passThroughArgT@H@@YAHH@Z}}
```

Comment:
I think this will break on windows, you need to escape `$`.

Reply:
Nice catch!